pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LocalBrokerDataTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.pulsar.broker.loadbalance;
```

Comment:
The new test class needs to add an apache license header. You can copy it from other classes.

Reply:
done.